### ISSUE TITLE
Empty CSV value does not output `null` in SQL output.

### ISSUE THREAD

**Issue by kherrera-ebsco** [kherrera-ebsco/cts#2]
## Problem

The tool does not convert empty CSV values into `null` in the SQL output.

### example.csv

```
alpha,beta,,delta
```

### example.sh

```
cts -i=Example \
    -m=0:y:one \
    -m=1:y:two \
    -m=2:y:three \
    -m=3:y:four \
    -- \
    example.csv
```

### example.sql

```sql
INSERT INTO Example (one, two, three, four)
     VALUES ('alpha', 'beta', '', 'delta')
```

## Resolution

For the above example to instead output the following:

```sql
INSERT INTO Example (one, two, three, four)
     VALUES ('alpha', 'beta', null, 'delta')
```

**Comment by kherrera-ebsco** [comment]
Closure was premature, CSV value is `None` but `Sql` outputting empty string.

**Comment by kherrera-ebsco** [comment]
Everything checks out. I was using `cts 0.1.0` instead of `cts 0.1.1`. The package published for `0.1.1` also does not correctly print the current version number. Package `0.1.1` accidentally reports as `0.1.0`. This will be fixed when another version is released.
